### PR TITLE
⬇️  pytest: install and configure Pytest

### PR DESCRIPTION
**Pull request by shlior7**
added pytest and pytest-django to pipfile
created pytest ini config file

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = Xpense.settings
+django_debug_mode = true
+python_files = tests.py test_*.py *_tests.py
```
Comment (adiabramovitch) [inline pytest.ini:4]: A newline at end of the file is missing

**Comment by adiabramovitch** [comment]
Maybe you should add configuration to run PyTest tests in GitHub actions as well.

File: pytest.ini (hunk 1 above)
Comment (ronyyosef) [reply pytest.ini:4]: #35

File: Pipfile [hunk 2]
```diff
@@ -5,9 +5,13 @@ name = "pypi"
 
 [packages]
 django = "*"
+pytest = "*"
```
Comment (ronyyosef) [inline Pipfile:9]: We don't need it in packages
packages are for things we need only in a production environment

**Review by LiorKGOW — CHANGES_REQUESTED** [review]
look at @ronyyosef  's comment
besides that looks good to me 👍

File: pytest.ini (hunk 1 above)
Comment (svatares) [reply pytest.ini]: @adiabramovitch @shlior7 was this resolved?

**Comment by adiabramovitch** [comment]
Please rebase

**Pull request closed without merging**